schedule/ha/bv/pvm_ha_priority_fencing.yaml

```diff
@@ -54,6 +56,10 @@ conditional_schedule:
     HA_CLUSTER_INIT:
       yes:
         - ha/barrier_init
+  grub_test:
+    STAGING:
```

Comment:
Can I know why you adding this conditional schedule for STAGING tests only?

Reply:
For staging test case will be failed if no this test module.

Reply:
I suppose this module should apply to every tests [even some of them don't have `grub timeout=-1`] :) but anyway, LGTM

Reply:
The reason why you have to add this test module is the below setting in agama profile 

```
  bootloader: {
    stopOnBootMenu: true
  },
```